### DISCUSSION TITLE
Why is devtool different in gen-webpack.config.js and gen-webpack.node.config.js

### DISCUSSION THREAD

**Discussion by meisenbarth-work** [eclipse-theia/theia#13914] [Q&A]
When packing our application for electron I realized the devtool for the backend and the frontend is different. Is there a reason the source maps are only omitted for the backend application? Should I keep the source maps for the frontend or could I remove them in my webpack.config.js?

devtool: mode === 'development' ? 'source-map' : false, <-> devtool: 'source-map',

**Comment by msujew** [comment]
Hey @meisenbarth-work,

the backend bundling is a relatively recent feature (see https://github.com/eclipse-theia/theia/pull/12412), while the frontend bundling has been there since the beginning of time. I assume it was just always there and noone ever actually cared until now.

It's probably reasonable to adjust this to omit source maps on production builds. Can you create a new issue to track this?

**Reply by meisenbarth-work** [reply]
Thanks for the fast reply.
Sure, will do.
